src/main/java/com/iridium/iridiumskyblock/utils/CheckEnvironment.java

```diff
@@ -0,0 +1,18 @@
+package com.iridium.iridiumskyblock.utils;
+
+public class CheckEnvironment {
```

Comment:
```suggestion
public class EnvironmentChecker {
```
This, but read the comment below first